install/docker/base

```diff
@@ -14,13 +14,12 @@
 
 FROM ubuntu:18.04
 
-# Expose our source so we can read in dependencies.
-ADD . /forseti-security/
-WORKDIR /forseti-security/
+# Add only the apt dependencies to avoid rebuilding base during development
```

Comment:
This is a super tiny nit, but if we are going to comment about avoid rebuilding during development, we should move it out of this comment line and put it at the top-level of this file.

```
# Do not add forseti source in this file to avoid rebuilding base during development.

FROM ubuntu:18.04

# Add the apt dependencies.
ADD install/dependencies/apt_packages.txt /tmp
```


```

Reply:
Makes sense to me, fixed